### PR TITLE
Use router

### PR DESCRIPTION
**Pull request by futurepaul**
this PR gets rid of solid-start and instead just relies on solid-router and solid-meta 

https://github.com/solidjs/solid-router
https://github.com/solidjs/solid-meta

### CODE REVIEW

**Comment by futurepaul** [comment]
all tests pass because I marked the restore test as `@slow` and filtered it out.

File: e2e/routes.spec.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        // Currency
+        await checkRoute(page, "/settings/currency", "Currency", checklist);
+        await page.goBack();
+
+        // Servers
+        await checkRoute(page, "/settings/servers", "Servers", checklist);
+        await page.goBack();
+
+        // Connections
+        await checkRoute(page, "/settings/connections", "Wallet Connections", checklist);
```
Comment (TonyGiorgio) [inline e2e/routes.spec.ts:80]: Can you test the wallet connections page with query parameters too? Like you did with the gift one below.

**Review by TonyGiorgio — CHANGES_REQUESTED** [review]

**Comment by TonyGiorgio** [comment]
Chrome and Safari look good to me. Still want to test android/ios pwa and then the native apps.

File: e2e/routes.spec.ts (hunk 1 above)
Comment (futurepaul) [reply e2e/routes.spec.ts]: added a test for wallet connections with params

**Comment by TonyGiorgio** [comment]
Tests failing

**Review by TonyGiorgio — COMMENTED** [review]
Tested pretty much every page on iOS and Android native & pwa. Just need the tests to pass reliably and I'd say it's good to merge whenever.

**Comment by futurepaul** [comment]
the one that failed was the send / receive test which hasn't failed before... could re-run again, but I feel pretty good about it. ran really quick locally

**Review by TonyGiorgio — APPROVED** [review]
Ran it 5 times and it passed. Nice work!

**Pull request merged**